<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_login">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_delete_board">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>글삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_delete_cmt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>댓글삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_logout">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>로그아웃</string>
    </property>
   </widget>
   <widget class="QLabel" name="state_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>상태</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_id">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
